src/battle_ai_util.c

```diff
@@ -2017,28 +2017,9 @@ bool32 HasSleepMoveWithLowAccuracy(u32 battlerAtk, u32 battlerDef)
     return FALSE;
 }
 
-bool32 IsHealingMoveEffect(u32 effect)
+bool32 IsHealingMove(u32 move)
```

Comment:
Would it not make sense to just remove the function and check the flag directly? It seems redundant to call it if you only check the flag

Reply:
I left the function, because I find writing `IsHealingMove(move)` faster/more aesthetic than `gBattleMoves[move].healBlockBanned`.